src/mqueryfront/src/Navigation.js

```diff
@@ -42,6 +42,13 @@ class Navigation extends Component {
                                 Recent jobs
                             </Link>
                         </li>
+                        {process.env.NODE_ENV == "development" ? (
+                            <li className="nav-item">
+                                <Link className="nav-link" to={"/storage"}>
+                                    Storage
+                                </Link>
+                            </li>
+                        ) : undefined}
```

Comment:
I'm not expert on react, but it's a very bad practice to return `undefined` in JS. Maybe `""`?

Reply:
"" means empty `<span></span>`, so it's not the same. Maybe `null` is better?